### PR TITLE
Add config for chunkloading time

### PR DESCRIPTION
**Pull request by someaddons**
# Changes proposed in this pull request:
- Add config for chunkloading time

Review please

### CODE REVIEW

File: src/main/resources/assets/minecolonies/lang/manual_en_us.json [hunk 1]
```diff
@@ -49,6 +49,9 @@
   "minecolonies.config.forceloadcolony": "Chunk Load Colony",
   "minecolonies.config.forceloadcolony.comment": "If part of the colony is loaded by an owner/officer, should the colony be kept loaded? (Set how many chunks are loaded with the \"Colony Chunk Loading Strictness\" option.)",
 
+  "minecolonies.config.loadtime": "Colony chunk loading duration",
```
Comment (Raycoms) [inline src/main/resources/assets/minecolonies/lang/manual_en_us.json:52]: (in minutes)

Comment (someaddons) [reply src/main/resources/assets/minecolonies/lang/manual_en_us.json:52]: well the comment mentions its default 10 mins

File: src/main/java/com/minecolonies/coremod/colony/Colony.java [hunk 2]
```diff
@@ -366,7 +366,9 @@ protected Colony(final int id, @Nullable final Level world)
         }
         this.permissions = new Permissions(this);
         researchManager = new ResearchManager(this);
-        colonyStateMachine = new TickRateStateMachine<>(INACTIVE, e -> {});
+        colonyStateMachine = new TickRateStateMachine<>(INACTIVE, e ->
+        {
+        });
```
Comment (uecasm) [inline src/main/java/com/minecolonies/coremod/colony/Colony.java:371]: This reformat seems kinda unnecessary.

Comment (someaddons) [reply src/main/java/com/minecolonies/coremod/colony/Colony.java:371]: autoformat

**Review by marchermans — APPROVED** [review]

**Pull request merged**